apps/src/authoredHints.js

```diff
@@ -77,20 +70,14 @@ module.exports = AuthoredHints;
  * @return {AuthoredHints[]}
  */
 AuthoredHints.prototype.getUnseenHints = function () {
-  var hints = this.contextualHints_.concat(this.hints_ || []);
-  return hints.filter(function (hint) {
-    return hint.alreadySeen === false;
-  });
+  return this.studioApp_.reduxStore.getState().authoredHints.unseenHints || [];
```

Comment:
Do you need the `|| []` here and below? I thought setting redux initial state would ensure that they're always defined.